local/gugcat/renderer.php

```diff
@@ -139,21 +138,41 @@ public function display_add_grade_form($course, $student, $gradeversions) {
         foreach($gradeversions as $gradeversion){
             $html .= html_writer::start_tag('div', array('class'=>'form-group row'));
             $html .= html_writer::start_tag('div', array('class'=> 'col-md-3'));
-            if ($gradeversion->itemname == get_string('moodlegrade', 'local_gugcat'))
-                $html .= html_writer::tag('label', $gradeversion->itemname. date(" [j/n/Y]", strtotime(userdate($gradeversion->timemodified))));
-            else 
-                $html .= html_writer::tag('label', $gradeversion->itemname);
```

Comment:
make sure to pull the latest from feature-gcat38 first before pushing your code.